src/components/Nav/Mobile/ExpandIcon.tsx

```diff
@@ -17,14 +16,9 @@ type ExpandIconProps = {
 }
 
 const ExpandIcon = ({ isOpen }: ExpandIconProps) => (
-  <Icon
+  <svg
     viewBox="0 0 24 25"
-    width={6}
-    height={6}
-    position="relative"
-    strokeWidth="2px"
-    display="inline-block"
-    stroke="currentColor"
+    className="relative inline-block h-6 w-6 stroke-current stroke-2 group-hover:text-primary-hover"
```

Comment:
<img width="515" alt="image" src="https://github.com/user-attachments/assets/947e3544-8e03-449a-8452-bf73c24427ed">

Is `group-hover:text-primary-hover` correct? Production does not use the primary color here on hover/pressing

Reply:
So, this is prod:
https://github.com/user-attachments/assets/d223a229-0c69-46a9-ad9a-efa5226c2656

Anyway, now that I think, it doesn't really matter, this is a mobile only menu where we are not going to have hover events. So, happy to remove it to make it cleaner.